### PR TITLE
[DO NOT DELETE] [MOD-667][internal] Support creating benchmarks runs

### PR DESCRIPTION
**Pull request by whilefalse**
# Problem

We need to support starting a benchmarks run from darwin-py.

# Solution

A new command, which can be used like this:

```bash
darwin benchmarks run --target v7/darwin-py-benchmarks-test --name FooBar --format darwin /path/to/annotations
```

### CODE REVIEW

**Review by balysv — APPROVED** [review]

File: darwin/cli.py [hunk 1]
```diff
@@ -167,6 +167,10 @@ def _run(args: Namespace, parser: ArgumentParser) -> None:
                 args.w,
                 args.h,
             )
+    elif args.command == "benchmarks":
```
Comment (owencjones) [inline darwin/cli.py:170]: Is there to be any other benchmarks command - if not `benchmarks-run` might be more appropriate, but if it's just future proofing, np.

File: darwin/importer/importer.py [hunk 2]
```diff
@@ -52,6 +54,16 @@
 """
 
 
+@dataclass
```
Comment (owencjones) [inline darwin/importer/importer.py:57]: Similar to something we had in mind anyway, so handy to have, cheers

File: tests/darwin/cli_functions_test.py [hunk 3]
```diff
@@ -189,10 +204,170 @@ def error_mock():
...
-
                         delete_files(dataset_identifier, ["one.jpg", "two.jpg"], True)
 
                         get_remote_dataset_mock.assert_called_once_with(dataset_identifier=dataset_identifier)
                         fetch_remote_files_mock.assert_called_once_with({"filenames": ["one.jpg", "two.jpg"]})
                         mock.assert_called_once()
                         exception.assert_called_once_with(1)
+
+
+def describe_benchmarks():
```
Comment (owencjones) [inline tests/darwin/cli_functions_test.py:215]: Good stuff, not enough things get tests added here.

**Review by owencjones — APPROVED** [review]
LGTM, liaise with QA about how this affects main use of importer mainly - feel free to point people my way

File: darwin/cli.py (hunk 1 above)
Comment (whilefalse) [reply darwin/cli.py:170]: I don't think there are any immediate plans, but I can imagine something like `benchmarks status <id>` or `benchmarks get-results <id>`.

**Pull request closed without merging**